frontend/appflowy_flutter/lib/plugins/ai_chat/application/chat_select_sources_cubit.dart

```diff
@@ -64,12 +72,32 @@ class ChatSource {
     return null;
   }
 
+  void resetIgnoreViewTypeRecursive({required String chatId}) {
+    ignoreStatusNotifier.value = view.id == chatId
+        ? IgnoreViewType.hide
+        : view.layout.isDocumentView
+            ? IgnoreViewType.none
+            : IgnoreViewType.disable;
```

Comment:
Extract into a helper method:
```
IgnoreViewType _ignoreTypeFromView(ViewPB view, String chatId) {
  if (chatId == view.id) {
    return IgnoreViewType.hide;
  }

  if (view.layout.isDocumentView) {
    return IgnoreViewType.none;
  }

  return IgnoreViewType.disable;
}
```
Because you also reuse this logic on line 181

Reply:
I now completely hide the chat stuff, including the current chat page. So this is just a single ternary now: 

``` 
ignoreStatus: view.layout.isDocumentView
          ? IgnoreViewType.none
          : IgnoreViewType.disable,
```